File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxLength = 0;

        int currentNum = 0;
        for(int i = 0; i < nums.length; i++){
            if(nums[i] == 1){
                currentNum += 1;
            } else {
                currentNum = 0;
            }
            if(currentNum > maxLength) {
                maxLength = currentNum;
            }
        }
        return maxLength;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        
        if k == len(nums):
            return nums 
            
        hashmap = defaultdict(int)
        
        for num in nums:
            hashmap[num] += 1

        sortedNums = sorted(hashmap, key = hashmap.get, reverse = True)
        return sortedNums[:k] 

            